components/ModTopbar.vue

```diff
@@ -1,18 +1,19 @@
 <template>
     <!-- styling will change from px to rm -->
-    <div class="h-[76px] bg-emerald-100">
-        <div v-if="store.activeScreen === ModerationScreen.dashboard ">
+    <div class="h-[76px] w-full">
+        <div v-if="store.activeScreen === ModerationScreen.dashboard">
             <p class="text-xl font-bold">TOPBAR MODERATION DASHBOARD</p>
         </div>
-        <div v-else-if="store.activeScreen === ModerationScreen.editSubmission">
-            <p class="text-xl font-bold">TOPBAR REVIEW SUBMISSION</p> 
+        <div class="h-[76px] w-full" v-else-if="store.activeScreen === ModerationScreen.editSubmission">
+            <ModEditSubmissionTopbar data-test-id="mod-edit-submission-topbar" submissionId="MattIsAGreatHumanBeing" />
```

Comment:
I think we can initialize `submissionId` as a variable with an empty string below.